src/components/tutorial/FeedbackButton.astro

```diff
@@ -46,7 +46,7 @@ const instanceId = Math.round(Math.random() * 10e9).toString(36);
 				</svg>
 			</button>
 			<form>
-				<h1><UIString key="feedback.formTitle" /></h1>
+				<h4><UIString key="feedback.formTitle" /></h4>
```

Comment:
I think SEO-wise, the H1 should actually be valid, since it's inside a from in a dialog

Reply:
I see what you mean about the context of the modal it makes sense, but I believe headers are meant to be considered globally. I only know this because having multiple `h1` elements on the page confuses search engines like google, and is explicitly flagged as an error in audit tools like ahref.

Reply:
In that case; Keep the change you made!

Reply:
I know Chris is OOO, but Chris' instinct was that this existing `<h1>` was actually fine too. I'm not the person to know, but just advising! Merge at your own risk!

Reply:
> I only know this because having multiple h1 elements on the page confuses search engines like google

I am pretty sure this isn't the truth anymore. Here's [confirmation from Google](https://www.youtube.com/watch?v=WsgrSxCmMbM) and a [helpful article on this matter](https://www.greenlanemarketing.com/resources/articles/seo-101-seo-and-multiple-h1-tags/).

Reply:
TIL! That's fascinating, okay will revert this then, I like the semantic argument.